spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/sending-messages.adoc

```diff
@@ -72,6 +72,19 @@ If the topic is configured to use `LOG_APPEND_TIME`, the user-specified timestam
 The `metrics` and `partitionsFor` methods delegate to the same methods on the underlying javadoc:org.apache.kafka.clients.producer.Producer[].
 The `execute` method provides direct access to the underlying javadoc:org.apache.kafka.clients.producer.Producer[].
 
+[NOTE]
+====
+When sending messages from Spring components, avoid using `@PostConstruct` methods
+if you rely on automatic topic creation via `NewTopic` beans. `@PostConstruct` runs
+before the application context is fully ready, which may cause
+`UnknownTopicOrPartitionException` on clean brokers.
```

Comment:
All good, but, please, consider to fix your change according to best practice rules we try to follow: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line.

It also would be great to make the text as "personless".
Use the language which would not be like pointing to someone.
In other words try to avoid "you", "we", "they" etc.